src/wp-includes/media-template.php

```diff
@@ -373,6 +373,15 @@ function wp_print_media_templates() {
 	<?php // Template for the Attachment Details two columns layout. ?>
 	<script type="text/html" id="tmpl-attachment-details-two-column">
 		<div class="attachment-media-view {{ data.orientation }}">
+			<?php
+			$deprecated_warning = '';
+			if ( isset( $_GET['error'] ) ) {
+				$deprecated_warning = (string) wp_unslash( $_GET['error'] );
+			}
+			if ( 'throw-depreciated-media.php' === $deprecated_warning ) {
```

Comment:
```suggestion
			if ( isset( $_GET['error'] ) && 'deprecated-file-media-php' === $_GET['error'] ) {
```

No need for two conditionals imho. Also seems no need to do wp_unslash() as the string will not be slashed and is not translatable.

Reply:
Ok! Fixed in updated PR.